zcash_client_backend/src/data_api/error.rs

```diff
@@ -45,9 +42,6 @@ pub enum Error<DataSourceError, CommitmentTreeError, SelectionError, FeeError> {
     /// No account could be found corresponding to a provided spending key.
     KeyNotRecognized,
 
-    /// No account with the given identifier was found in the wallet.
-    AccountNotFound(AccountId),
-
```

Comment:
See https://github.com/zcash/librustzcash/pull/1175/files#r1515352198 for why I'm confused that this is being removed.

Reply:
I removed it because no code uses it, and because it was using zip32::AccountId, which is inappropriate anyway.